Thanks. There's a little pylinting to do:
```
Run pylint $( find . -path './adafruit*.py' )
************* Module adafruit_led_animation.animation.comet
adafruit_led_animation/animation/comet.py:51:0: R0902: Too many instance attributes (13/11) (too-many-instance-attributes)
adafruit_led_animation/animation/comet.py:51:0: R0902: Too many instance attributes (13/11) (too-many-instance-attributes)
adafruit_led_animation/animation/comet.py:51:0: R0902: Too many instance attributes (13/11) (too-many-instance-attributes)
adafruit_led_animation/animation/comet.py:51:0: R0902: Too many instance attributes (13/11) (too-many-instance-attributes)
adafruit_led_animation/animation/comet.py:51:0: R0902: Too many instance attributes (13/11) (too-many-instance-attributes)
adafruit_led_animation/animation/comet.py:51:0: R0902: Too many instance attributes (13/11) (too-many-instance-attributes)
************* Module adafruit_led_animation.color
adafruit_led_animation/color.py:85:4: R1705: Unnecessary "elif" after "return" (no-else-return)
```

--- a/adafruit_led_animation/animation/comet.py
+++ b/adafruit_led_animation/animation/comet.py
@@ -62,7 +62,7 @@ class Comet(Animation):
     :param bool bounce: Comet will bounce back and forth. Defaults to ``True``.
     """
 
-    # pylint: disable=too-many-arguments
+    # pylint: disable=too-many-arguments,too-many-instance-attributes
     def __init__(
         self,
         pixel_object,

--- a/adafruit_led_animation/color.py
+++ b/adafruit_led_animation/color.py
@@ -88,7 +88,7 @@ def calculate_intensity(color, intensity=1.0):
             int(color[1] * intensity),
             int(color[2] * intensity),
         )
-    elif len(color) == 4 and isinstance(color[3], float):
+    if len(color) == 4 and isinstance(color[3], float):
         return (
             int(color[0] * intensity),
             int(color[1] * intensity),

